server/core/templates/credentials_reset.html

```diff
@@ -5,14 +5,13 @@
 (% endblock %)
 
 (% block body %)
-<div class="d-flex align-items-start form-cred-reset-body">
-    <main class="w-100">
-        <div class="py-3 text-center">
-            <h3>Updating Credentials</h3>
-            <p>(( names ))</p>
-            <p>(( domain_info.display_name() ))</p>
-        </div>
-        (( credentials_update_partial|safe ))
-    </main>
-</div>
+<main class="form-cred-reset-body container my-auto">
+    <div class="text-center">
+        <h3>Updating Credentials</h3>
+        <p>(( names ))</p>
+        <p>(( domain_info.display_name() ))</p>
```

Comment:
This is a little weird to see from a user perspective, can we add some labels for the fields?

Same as what's in `server/core/templates/credentials_status.html`